openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -303,7 +303,7 @@ Wallet implementations using the key attestation format specified in Annex D of
 
 # Security Considerations {#security_considerations}
 
-The security considerations in [@!OIDF.OID4VCI] and [@!OIDF.OID4VP] apply.
+Note that security considerations for OpenID for Verifiable Credential Issuance are defined in [@!OIDF.OID4VCI] and for OpenID for Verifiable Presentations in [@!OIDF.OID4VP].
```

Comment:
```suggestion
Note that security considerations for OpenID for Verifiable Credential Issuance are defined in Section 13 of [@!OIDF.OID4VCI] and for OpenID for Verifiable Presentations in Section 14 of [@!OIDF.OID4VP].
```
i think this reads nicer..

Reply:
I'd rather keep it as is but if we do this then we need to mention the DC API security considerations too:

```suggestion
Note that security considerations for OpenID for Verifiable Credential Issuance are defined in Section 13 of [@!OIDF.OID4VCI] and for OpenID for Verifiable Presentations in Section 14 (for redirect based flows) or Section A.5 (for DC API) of [@!OIDF.OID4VP].
```

Reply:
i like it - thanks for the suggestion